src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -298,9 +312,8 @@ export class SearchSource {
   /**
    * Returns body contents of the search request, often referred as query DSL.
    */
-  async getSearchRequestBody() {
-    const searchRequest = await this.flatten();
-    return searchRequest.body;
+  getSearchRequestBody() {
```

Comment:
Why is this not async any more?

Reply:
flatten was never async, this is just ceanup